pkg/pod/status.go

```diff
@@ -408,6 +413,34 @@ func isPodHitConfigError(pod *corev1.Pod) bool {
 	return false
 }
 
+// isPullImageError returns true if the Pod's status indicates there are any error when pulling image
+func isPullImageError(pod *corev1.Pod) bool {
+	for _, containerStatus := range pod.Status.ContainerStatuses {
+		if containerStatus.State.Waiting != nil && isImageErrorReason(containerStatus.State.Waiting.Reason) {
+			return true
+		}
+	}
+	return false
+}
+
+func isImageErrorReason(reason string) bool {
+	// Reference from https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/images/types.go#L26
```

Comment:
would you mind using a permalink here?

Reply:
Sorry, what do you mean about permalink? I guess you mean it is not a  good choice put the link here?

Maybe putting the link in the commit message is a better way?  Putting the message in the comment or the commit message is the only two ways  I can think of. 

If you have a better choice, I‘d love to use it. 😁